Scalar.FunctionalTests/Program.cs

```diff
@@ -113,6 +113,10 @@ public static void Main(string[] args)
                 // This is non-critical, and watchman may not be installed.
             }
 
+            // TODO Do we need to iterate over all of the repos that we created and
+            // TODO issue a `git fsmonitor--daemon --stop` on each?  Note that we
+            // TODO will have code in DeleteEnlistment() to do that during the tests.
+
```

Comment:
The `DeleteEnlistment()` should be sufficient, I think. If we have problems, then we can try to do something here.